### ISSUE TITLE
support grid-area, grid-row-end, and grid-column-end with span syntax

### ISSUE THREAD

**Issue by hellochar** [postcss/autoprefixer#883]
autoprefixer seems to completely ignore `grid-area`, `grid-row-end`, and `grid-column-end`, even though the `span` syntax should be translatable:

```css
grid-area: 5 / 1 / span 1 / span 5;

/* could turn into */

-ms-grid-row: 5;
-ms-grid-column: 1;
-ms-grid-row-span: 1;
-ms-grid-column-span: 5;
```

**Comment by ai** [comment]
Sure. Do you want to make PR? I will not have on this week.

**Comment by ai** [comment]
https://github.com/postcss/autoprefixer/pull/938 improved Grid support

**Comment by ai** [comment]
https://github.com/postcss/autoprefixer/pull/943 improved Grid support even better

**Comment by ai** [comment]
Even `grid-template-areas` support was added https://github.com/postcss/autoprefixer/pull/948

**Comment by ai** [comment]
Released in 7.2
